scripts/jest.js

```diff
@@ -1,5 +1,5 @@
 const jest = require('jest');
-const config = require('./ui_framework_test_config');
```

Comment:
Alright, @cjcenizal and I hashed it out:

- This files should be moved to `src/jest/cli.js`
- `scripts/jest.js` should look exactly like the rest of the scripts in `scripts/` and just require babel followed by `src/jest/cli.js`